### DISCUSSION TITLE
Cards layout :  Change filetype by mediaduration managed property

### DISCUSSION THREAD

**Discussion by tocri** [microsoft-search/pnp-modern-search#3729] [Q&A] [answered]
Hi, 
Is it possible to change the filetrype property to mediaduration property ?
I tried to change the filetype properties to mediaduration, i changed also to mediaduration in the handlebar and i changed the css to mediaduration but the file icon is still there.

Is it possible to do this  ? 

thanks

**Comment by wobba** [comment]
Would be helpful if you add some more information and details on what you are trying to achieve. But yes, you can add MediaDuration to the requested properties and use it.

**Comment by tocri** [comment]
Hi  @wobba , sorry for the delay. I was in NYC.

I try to replace the filetype icon by the value of meidaduration.
![image](https://github.com/microsoft-search/pnp-modern-search/assets/10110260/b4f29a2b-3801-46f3-b2ac-655062b99d26)

I woud like to achieve this
<img width="227" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/10110260/763c1033-210b-421d-8416-db70651afe86">

I know mediaduration displays the value in secondes but my purpose is to check if is it possible to replace filetype. then i will make a flow that converts mediaduration in minutes.

**Comment by tocri** [comment, marked as answer]
@wobba @FranckyC 

Maybe i'm wrong but aftet many attemps, it seems it's no be possible to change the filetype icon by the duration of the media like 
![image](https://github.com/microsoft-search/pnp-modern-search/assets/10110260/184bdf86-4647-4536-be9b-2afe0ba64194)

Could you confirm ?

**Reply by wobba** [reply]
These cards are a bit limited and you should rather create your own html template.

**Reply by tocri** [reply]
@wobba I'm not an expert in modern-search, so i have resolved my issue by added the duration on the localization slot 

![image](https://github.com/user-attachments/assets/2d2ff891-b03d-4baa-95f0-2ddbff72cd75)
